### DISCUSSION TITLE
Switching to TlmPacketizer

### DISCUSSION THREAD

**Discussion by MichaelStarks** [nasa/fprime#1665] [Q&A]
Hello,

I would like to switch from Svc::TlmChan to Svc::TlmPacketizer. I have searched my workspace and changed every instance of the TlmChan object, named chanTlm, to the TlmPacketizer object, named packetTlm, and the TlmChan type to TlmPacketizer. I am able to build the workspace but when I run the executable, I immediately get a Seg Fault. Am I missing any steps to get TlmPacketizer working?

**Comment by timcanham** [comment]
There is a tool we have to integrate with the build system so you can specify packets and use them with `TlmPacketizer`. It's not there yet, but it's on our short list!
